### DISCUSSION TITLE
Role Management

### DISCUSSION THREAD

**Discussion by FrancyMak** [BG-Software-LLC/SuperiorSkyblock2#625] [Denied Feature Requests] [answered]
I would like the rank system to be fixed, currently the placeholder %superior_player_role% does not show the displayname but shows the name that is assigned within the config. 

On top of that if I try to change the role names, they are not updated within the server and sometimes it results in the config.yml being reset.

If the config file is this, why shouldn't it saved in the placeholder?

` 
 Capo:
      id: 3
      name: Capo
      weight: 3
      permissions:
      - ALL
 `
 
 ![immagine_2021-09-26_143106](https://user-images.githubusercontent.com/75610581/134808102-1e3dcc24-4768-4ea8-8dbf-3bab9b1d8b85.png)

**Comment by OmerBenGera** [comment, marked as answer]
Please open an issue on github regarding this.
It sounds like a bug to me.
